pkg/db/migration.go

```diff
@@ -52,12 +52,8 @@ func (d *DatabaseMigrator) Migrate(ctx context.Context) error {
 }
 
 func getStatikSrc() (source.Driver, error) {
```

Comment:
Maybe we can rename it, now that statik is out?

Reply:
good idea - updated, thanks.